gitpod-io/website
# :sparkles: evolve Gitpod Teams to Gitpod Orgs

## Description

Related: 
- https://github.com/gitpod-io/gitpod/issues/15980
- https://github.com/gitpod-io/gitpod/pull/16050

[_Internal Conversation_](https://gitpod.slack.com/archives/C01KGM9ETU6/p1674739905075359) 🧵 

Also, existing Routes are handled and redirected to new docs pages

## How to test



This PR is relatively hard to test for all. But, mainly check the prior important pages:

- [`/pricing`](https://deploy-preview-3314--gitpod-kumquat.netlify.app/pricing) (pricing calculator, feature table, top card on pricing page)
- Left Menu Items Section of Docs ("Organizations" is a new one under "Configure" Category)
- [`/docs/configure/orgs`](https://deploy-preview-3314--gitpod-kumquat.netlify.app/docs/configure/orgs)
- [`/docs/configure/billing#configure-organization-billing`](https://deploy-preview-3314--gitpod-kumquat.netlify.app/docs/configure/billing#configure-organization-billing) (Currently changes text only, TODO: Images will create a follow-up PR with new screenshots, once https://github.com/gitpod-io/gitpod/pull/16050 will be deployed)
- [`/docs/configure#organizations`](https://deploy-preview-3314--gitpod-kumquat.netlify.app//docs/configure#organizations)
- [`/docs/configure/billing/org-plans`](https://deploy-preview-3314--gitpod-kumquat.netlify.app/docs/configure/billing/org-billing)
- [`/docs/configure/billing/org-plans`](https://deploy-preview-3314--gitpod-kumquat.netlify.app/docs/configure/billing/org-plans)


## Release Notes



```release-note
Gitpod Teams are Gitpod Organization now
```
